src/PatronAccessDetail/PatronAccessDetail.css

```diff
@@ -0,0 +1,19 @@
+.access {
+  padding: 5px;
+  border-radius: 6px;
+  line-height: var(--line-height);
+}
+
+.allowed {
+  background-color: var(--success-fill);
+  border: 1px solid var(--success);
+}
+
+.notAllowed {
+  background-color: var(--warn-fill);
+  border: 1px solid var(--warn);
+}
+
+.marginRight{
```

Comment:
Is it possible to use css class names that related to element on the page instead of their values?
For example `iconWrapper` or something like that.

Reply:
I think this class name makes sense.